### ISSUE TITLE
java.lang.IllegalArgumentException when running 5.1.0

### ISSUE THREAD

**Issue by vewert** [dropbox/dropbox-sdk-java#371]
I tried using version 5.1.0 in my Java Desktop application, and when I try to run it, I get the following:

```
> Task :run FAILED
Inside run
Error occurred during initialization of boot layer
java.lang.module.FindException: Unable to derive module descriptor for C:\Users\vewert\.gradle\caches\modules-2\files-2.1\org.khronos\opengl-api\gl1.1-android-2.1_r1\8827d279add29cf9115820671e7879de7bf80390\opengl-api-gl1.1-android-2.1_r1.jar
Caused by: java.lang.IllegalArgumentException: opengl.api.gl1.1.android: Invalid module name: '1' is not a Java identifier
```

Using 5.0.0 (and earlier) has no problem.

Using:
Java 17.0.1
Gradle 7.3.2 (for building and running)

**Comment by vewert** [comment]
I think I may have found the problem (at least for me). It seems the dependencies have changed, and `dropbox-core-sdk` now seems to depend on `android`, which in turn depends on `opengl-api`. Since I am using this for a Java desktop app, I don't need android, so in my gradle file I excluded the dependency on `android`:

```kotlin
implementation(group = "com.dropbox.core", name = "dropbox-core-sdk", version = "5.1.0") {
    exclude("com.google.android")
  }
```
I can now run the application without errors. Hopefully excluding `android` doesn't cause any problems.

The question now is, why did the dependencies change? Looking at the pom file (from maven central), a lot of the dependencies were changed from `<scope>provided</scope>` to `<scope>runtime</scope>`
